components/separator.json

```diff
@@ -1,8 +1,40 @@
 {
   "separator": {
     "comment": "",
-    "figma": "",
+    "figma": "https://www.figma.com/file/NhiWBKypDml4kDCTnLUygT/Component-Explorations?node-id=1242%3A166850",
     "primary": "@theme-outline-primary-normal",
-    "secondary": "@theme-outline-secondary-normal"
+    "secondary": "@theme-outline-secondary-normal",
+    "gradient-normal": {
```

Comment:
In the separator component, please add this figma file (https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/branch/WG6O99kWm33upbqMdnnIQA/Components---Windows%2BWeb?node-id=12652%3A3113) and leave primary and secondary as they are

Reply:
In a separate file inside `platformcomponents` (you can call this new file `composearea`, for example), put:
```
"composearea":{
     "comment": "This is applied to the compose area component inside messaging tab.",
     "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/branch/WG6O99kWm33upbqMdnnIQA/Components---Windows%2BWeb?node-id=29743%3A57187"
     "divider":{
             "default":{
                    "#normal": ["@theme-gradientdivider-normal-0", "@theme-gradientdivider-normal-1"],
                    "#hover": ["@theme-gradientdivider-hover-0", "@theme-gradientdivider-hover-1"]
            },
           "externalparticipant":{
                    "#normal": ["@theme-gradientdivider-warning-normal-0", "@theme-gradientdivider-warning-normal-1"],
                    "#hover": ["@theme-gradientdivider-warning-hover-0", "@theme-gradientdivider-warning-hover-1"]
            },
           "announcementspace":{
                    "#normal": ["@theme-gradientdivider-announcement-normal-0", "@theme-gradientdivider-announcement-normal-1"],
                    "#hover": ["@theme-gradientdivider-announcement-hover-0", "@theme-gradientdivider-announcement-hover-1"]
            },
           "classified":{
                    "#normal": ["@theme-gradientdivider-error-normal-0", "@theme-gradientdivider-error-normal-1"],
                    "#hover": ["@theme-gradientdivider-error-hover-0", "@theme-gradientdivider-error-hover-1"]
            },
           "multitypespace":{
                    "#normal": ["@theme-gradientdivider-error-normal-0", "@theme-gradientdivider-error-normal-1"],
                    "#hover": ["@theme-gradientdivider-error-hover-0", "@theme-gradientdivider-error-hover-1"]
            },
           "announcementlocked":{
                    "#normal": ["@theme-gradientdivider-normal-0", "@theme-gradientdivider-normal-1"],
                    "#hover": ["@theme-gradientdivider-hover-0", "@theme-gradientdivider-hover-1"]
            },
           "blockedpresence":{
                    "#normal": ["@theme-gradientdivider-normal-0", "@theme-gradientdivider-normal-1"],
                    "#hover": ["@theme-gradientdivider-hover-0", "@theme-gradientdivider-hover-1"]
            },
           "instantmessaggingonly":{
                    "#normal": ["@theme-gradientdivider-normal-0", "@theme-gradientdivider-normal-1"],
                    "#hover": ["@theme-gradientdivider-hover-0", "@theme-gradientdivider-hover-1"]
            },
           "peek":{
                    "#normal": ["@theme-gradientdivider-normal-0", "@theme-gradientdivider-normal-1"],
                    "#hover": ["@theme-gradientdivider-hover-0", "@theme-gradientdivider-hover-1"]
            },
     }
}
```